.gitignore

```diff
@@ -25,3 +25,4 @@ metadata/
 # avoid publishing translation config
 crowdin.yml
 /.editorconfig
+/cartridges/int_alma/package-lock.json
```

Comment:
Just before merge... are you sure you want ignore `package-lock.json` here  ? (and why  ?)